project_event/report/project_task_activity_report.xml

```diff
@@ -198,11 +198,20 @@
               <th>Resource</th>
             </tr>
             <t t-foreach="tasks_details" t-as="resource">
-              <tr>
-                <td> <t t-esc="resource['task']"/></td>
-                <td> <t t-esc="resource['resource_type']"/></td>
-                <td> <t t-raw="resource['resource']"/></td>
-              </tr>
+              <t t-if="resource['resource_type']=='Instrument'">
```

Comment:
It's good. resource['resource_type'] is taking the resource type title, this should be changed in the future. Now it works, but it would not work in another language than English or french